--- CMakeLists-orig.txt	2017-08-25 15:27:05.086456148 +1000
+++ CMakeLists.txt	2017-08-25 15:27:05.086456148 +1000
@@ -6,15 +6,6 @@
 #
 # ----------------------------------------------------------------------------
 
-# Disable in-source builds to prevent source tree corruption.
-if(" ${CMAKE_SOURCE_DIR}" STREQUAL " ${CMAKE_BINARY_DIR}")
-  message(FATAL_ERROR "
-FATAL: In-source builds are not allowed.
-       You should create separate directory for build files.
-")
-endif()
-
-
 include(cmake/OpenCVMinDepVersions.cmake)
 
 if(CMAKE_GENERATOR MATCHES Xcode AND XCODE_VERSION VERSION_GREATER 4.3)
