--- ncbi-blast-2.6.0+-src/c++/src/build-system/Makefile.in.top.orig
2017-06-27 16:27:34.725091206 +1000
+++ ncbi-blast-2.6.0+-src/c++/src/build-system/Makefile.in.top
2017-06-28 15:00:06.802995467 +1000
@@ -51,7 +51,8 @@
 	    done
 	cd $(includedir0) && find * -name CVS -prune -o -print |\
             cpio -pd $(pincludedir)
-	$(INSTALL) -m 644 $(incdir)/* $(pincludedir)
+	$(INSTALL) -m 644 $(incdir)/*.h $(pincludedir)
+	$(INSTALL) -m 644 $(incdir)/common/*.h $(pincludedir)/common
 ## set up appropriate build and status directories somewhere under $(libdir)?
 
 install-gbench:
