--- tk8.6.6/unix/tkWindow.c.orig	2017-06-29 15:57:14.000000000 +1000
+++ tk8.6.6/unix/tkWindow.c	2017-06-29 15:56:33.026558534 +1000
@@ -948,6 +948,7 @@
 
     Tcl_SetVar2(interp, "tk_patchLevel", NULL, TK_PATCH_LEVEL, TCL_GLOBAL_ONLY);
     Tcl_SetVar2(interp, "tk_version",    NULL, TK_VERSION,     TCL_GLOBAL_ONLY);
+    Tcl_SetVar2(interp, "tk_library",    NULL, TK_LIBRARY,     TCL_GLOBAL_ONLY);
 
     tsdPtr->numMainWindows++;
     return tkwin;
